docs/guide/performance.md

```diff
@@ -55,14 +55,14 @@ Barrel files are files that re-export the APIs of other files in the same direct
 
 ```js
 // src/utils/index.js
-export * from './color'
-export * from './dom'
-export * from './string'
+export * from './color.js'
```

Comment:
I think we should start using extensions in code example and docs so people get used to it :)